<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Абоненты</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cmdAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить абонента</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>add-icon.png</normaloff>add-icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>600</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Actions-document-edit-icon.png</normaloff>Actions-document-edit-icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdDel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>600</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Actions-edit-delete-icon.png</normaloff>Actions-edit-delete-icon.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Zoom-icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkName">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>180</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkFName">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFName">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>240</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPName_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPName">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>300</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPhone">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>120</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPhone">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер телефона</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdAvail">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>600</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Незанятые номера</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>my-numbers-icon.png</normaloff>my-numbers-icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdStat">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>600</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Статистика подключений</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>chart-icon.png</normaloff>chart-icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdShow">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkConType">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>340</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип абонента</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbConType">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>370</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkYear">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>420</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Год подключения</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtYear">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>640</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tblSubsc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>571</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="separator"/>
    <addaction name="mnuAdd"/>
    <addaction name="mnuEditSub"/>
    <addaction name="mnuDel"/>
   </widget>
   <widget class="QMenu" name="mnuTools">
    <property name="title">
     <string>Сервис</string>
    </property>
    <addaction name="mnuAvail"/>
    <addaction name="mnuStat"/>
    <addaction name="separator"/>
    <addaction name="mnuList"/>
   </widget>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="mnuOpen"/>
    <addaction name="mnuExit"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuEdit"/>
   <addaction name="mnuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Файл</string>
   </property>
  </action>
  <action name="mnuOpen">
   <property name="text">
    <string>Открыть базу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сохранить базу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="mnuExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="mnuAdd">
   <property name="text">
    <string>Добавить абонента</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="mnuEditSub">
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="mnuDel">
   <property name="text">
    <string>Удалить абонента</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="mnuAvail">
   <property name="text">
    <string>Незанятые номера</string>
   </property>
  </action>
  <action name="mnuStat">
   <property name="text">
    <string>Статистика подключений</string>
   </property>
  </action>
  <action name="mnuList">
   <property name="text">
    <string>Полный список абонентов</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
